src/components/StackList/StackList.jsx

```
@@ -0,0 +1,42 @@
+    )
+}
+
+export const StackList = ({traces}) => {
+
+    const generateStackList = () => {
+        const traceList = traces.map(trace => 
+            <TraceNode 
+                functionName={trace.functionName}
+                fileName={trace.fileName}
+                lineNumber={trace.lineNumber}             
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)